### ISSUE TITLE
Grammar check in Strings

### ISSUE THREAD

**Issue by mohitleo9** [rhysd/vim-grammarous#55]
Hey, 
Is there a way to force the checker to validate only 'strings' and 'comments' for grammar.

**Comment by mohitleo9** [comment]
Actually took a quick look, could https://github.com/rhysd/vim-grammarous/blob/8fbcc8e61409f93f215fe627b231e4550b967e18/autoload/grammarous.vim#L135-L138 be generalized to pass in any (or an array of) syntax group and provide predefined settings from strings and comments?

**Comment by rhysd** [comment]
Yeah, it can be extended as you're saying. I label this as enhancement.
